### PR TITLE
Issue 133 doidatasets primary key

### PR DESCRIPTION
**Pull request by kauberry**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Check List

- [ ] All tests pass. See <https://github.com/pacifica/pacifica-docs/blob/master/development/TESTING.md>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable

### CODE REVIEW

File: metadata/rest/test/test_orm.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def test_perf_of_large_insert(self):
                                data=txt_trans, headers=self.headers)
             self.assertEqual(req.status_code, 200)
         end_time = time()
-        self.assertTrue(end_time - start_time < 90)
```
Comment (dmlb2000) [inline metadata/rest/test/test_orm.py]: This should be reverted... or we should document justification for changing it

File: requirements-dev.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ mock
 nose
 peewee>2
 pep257
-psycopg2
```
Comment (dmlb2000) [inline requirements-dev.txt]: I don't think this is needed, why is this different from the normal package?

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 cherrypy
 elasticsearch
 peewee>2
-psycopg2
```
Comment (dmlb2000) [inline requirements.txt]: Same here...

File: requirements-dev.txt (hunk 2 above)
Comment (kauberry) [reply requirements-dev.txt]: They are renaming the package from psycopg2 to psycopg2-binary and deprecating the old name.

File: requirements.txt (hunk 3 above)
Comment (kauberry) [reply requirements.txt]: Same here

File: metadata/rest/test/test_orm.py (hunk 1 above)
Comment (kauberry) [reply metadata/rest/test/test_orm.py]: Rolling this one back. This test was failing multiple times, but I think it was just Travis misbehaving.

File: requirements-dev.txt (hunk 2 above)
Comment (dmlb2000) [reply requirements-dev.txt]: github.com/psycopg/psycopg2/issues/674 this bug has links to the discussion and plan moving forward. The package specified is going to be a source only build. Which is fine for travis.

**Pull request merged**